src/Console/CommandRunner.php

```diff
@@ -70,6 +71,10 @@ public function __construct(ConsoleApplicationInterface $app, $root = 'cake')
             '--help' => 'help',
             '-h' => 'help',
         ];
+
+        if ($app instanceof EventDispatcherInterface) {
```

Comment:
Shouldn't this also be the EventApplicationInterface?

Reply:
This check is made in later method when the events() method is called.